As a user, I don't expect to see an error indicating that I need to provide one or more summary statistics to calculate when my summary_statistics declaration already includes two stats

The complete declaration is below with the WRDS URL and label omitted. The `summary_statistics` are declared as follows:

```yaml
summary_statistics:
  statistics:
  - box plot
  - name: histogram
    bins: 10
``` 

The exception is:

```
wres.pipeline.InternalWresException: Could not complete project execution
	at wres.pipeline.Evaluator.evaluate(Evaluator.java:409)
	at wres.pipeline.Evaluator.evaluate(Evaluator.java:181)
	at wres.Functions.evaluate(Functions.java:135)
	at wres.server.EvaluationService.lambda$startEvaluation$2(EvaluationService.java:695)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: wres.pipeline.WresProcessingException: Encountered an error while processing evaluation 'cHBKsXzcEVghU_kh6CchNIXibS4': 
	at wres.pipeline.Evaluator.evaluate(Evaluator.java:856)
	at wres.pipeline.Evaluator.evaluate(Evaluator.java:380)
	... 7 common frames omitted
Caused by: java.lang.IllegalArgumentException: Provide one or more summary statistics to calculate.
	at wres.metrics.SummaryStatisticsCalculator.<init>(SummaryStatisticsCalculator.java:1041)
	at wres.metrics.SummaryStatisticsCalculator.of(SummaryStatisticsCalculator.java:151)
	at wres.pipeline.EvaluationUtilities.getSumStatsCalculators(EvaluationUtilities.java:1709)
	at wres.pipeline.EvaluationUtilities.getSumStatsCalculators(EvaluationUtilities.java:1590)
	at wres.pipeline.EvaluationUtilities.getSumStatsCalculators(EvaluationUtilities.java:264)
	at wres.pipeline.Evaluator.evaluate(Evaluator.java:755)
	... 8 common frames omitted
```

If this exception is expected, please explain why. If its not expected, then it is a bug; please fix it. I'll grab the data in a bit,

Hank

=========================

```yaml
label: 'LABEL OMITTED'
observed:
  label: USGS NWIS Stage Observations
  sources:
  - interface: usgs nwis
    uri: https://nwis.waterservices.usgs.gov/nwis/iv
  variable:
    name: '00065'
  feature_authority: usgs site code
  type: observations
predicted:
  label: WRDS AHPS API Stage Forecast
  sources:
  - interface: wrds ahps
    uri: https://OMITTED/api/rfc_forecast/v2.0/forecast/stage
  variable:
    name: HG
  feature_authority: nws lid
  type: single valued forecasts
unit: FT
lead_time_pools:
  period: 6
  frequency: 6
  unit: hours
lead_times:
  minimum: 0
  maximum: 72
  unit: hours
reference_dates:
  minimum: '2024-04-18T12:00:00Z'
  maximum: '2024-04-30T00:00:00Z'
valid_dates:
  minimum: '2024-04-18T12:00:00Z'
  maximum: '2024-04-30T00:00:00Z'
features:
- observed: '15056210'
  predicted: TYAA2
- observed: '15129120'
  predicted: ALEA2
- observed: '15056500'
  predicted: CKTA2
- observed: '15087700'
  predicted: IRVA2
- observed: '15052500'
  predicted: MNDA2
- observed: '15081497'
  predicted: SCKA2
- observed: '15129500'
  predicted: SRYA2
- observed: '15024800'
  predicted: STKA2
- observed: '15041200'
  predicted: TKUA2
- observed: '15564900'
  predicted: KRHA2
- observed: '15875000'
  predicted: CRUA2
- observed: '15744500'
  predicted: KIAA2
- observed: '15896000'
  predicted: KUPA2
- observed: '15908000'
  predicted: SGRA2
- observed: '15564879'
  predicted: SLAA2
- observed: '15747000'
  predicted: WULA2
- observed: '15348000'
  predicted: FMRA2
- observed: '15356000'
  predicted: YEAA2
- observed: '15453500'
  predicted: YRBA2
- observed: '15478000'
  predicted: BGDA2
- observed: '15514000'
  predicted: CHFA2
- observed: '15511000'
  predicted: CHLA2
- observed: '15493400'
  predicted: CRHA2
- observed: '15470000'
  predicted: CRNA2
- observed: '15515500'
  predicted: ENNA2
- observed: '15477740'
  predicted: GBDA2
- observed: '15518040'
  predicted: HNRA2
- observed: '15493700'
  predicted: MCDA2
- observed: '15484000'
  predicted: SALA2
- observed: '15485500'
  predicted: TAFA2
- observed: '15493000'
  predicted: UCHA2
- observed: '15275100'
  predicted: CHRA2
- observed: '15276000'
  predicted: SHIA2
- observed: '15238990'
  predicted: BRUA2
- observed: '15200280'
  predicted: GRSA2
- observed: '15209700'
  predicted: MXYA2
- observed: '15239900'
  predicted: APTA2
- observed: '15258000'
  predicted: COOA2
- observed: '15248000'
  predicted: MPTA2
- observed: '15271000'
  predicted: SIXA2
- observed: '15266110'
  predicted: SKLA2
- observed: '15243900'
  predicted: SNOA2
- observed: '15266300'
  predicted: SXRA2
- observed: '15304000'
  predicted: CJXA2
- observed: '15303900'
  predicted: KLCA2
- observed: '15303600'
  predicted: MCGA2
- observed: '15292000'
  predicted: GCSA2
- observed: '15290000'
  predicted: LSUA2
- observed: '15284000'
  predicted: MATA2
- observed: '15292800'
  predicted: MONA2
- observed: '15292780'
  predicted: SUNA2
- observed: '15292700'
  predicted: TRTA2
- observed: '15294005'
  predicted: WLWA2
feature_service:
  uri: https://OMITTED/api/location/v3.0/metadata
metrics:
- name: sample size
- name: mean error
- name: pearson correlation coefficient
- name: mean square error skill score
- name: box plot of errors
- name: root mean square error
- name: coefficient of determination
minimum_sample_size: 1
summary_statistics:
  statistics:
  - box plot
  - name: histogram
    bins: 10
duration_format: hours
output_formats:
- format: csv2
- format: pairs
- format: png
```

--- a/src/wres/pipeline/EvaluationUtilities.java
+++ b/src/wres/pipeline/EvaluationUtilities.java
@@ -250,7 +250,7 @@ class EvaluationUtilities
             return Map.of();
         }
 
-        // Collect the supported feature dimensions to aggregate
+        // Collect the supported dimensions to aggregate
         // Note that clearing threshold values is linked to these aggregation dimensions.
         Set<SummaryStatistic.StatisticDimension> dimensions =
                 declaration.summaryStatistics()
@@ -260,6 +260,12 @@ class EvaluationUtilities
                            .filter( d -> d != SummaryStatistic.StatisticDimension.RESAMPLED
                                          && d != SummaryStatistic.StatisticDimension.TIMING_ERRORS )
                            .collect( Collectors.toSet() );
+
+        if( dimensions.isEmpty() )
+        {
+            throw new IllegalStateException( "Summary statistics were requested but no summary statistics dimensions "
+                                             + "could be found." );
+        }
 
         return EvaluationUtilities.getSumStatsCalculators( declaration,
                                                            dimensions,
@@ -1522,7 +1528,7 @@ class EvaluationUtilities
             List<FeatureGroupFilterAdapter> filters =
                     EvaluationUtilities.getOneBigFeatureGroupForSummaryStatistics( declaration );
             featureFilters.addAll( filters );
-            LOGGER.debug( "Created {} filters for all geographic features.", filters.size() );
+            LOGGER.info( "Created {} filters for all geographic features.", filters.size() );
         }
 
         // Summary statistics across all geographic features in a single feature group

--- a/wres-config/src/wres/config/yaml/DeclarationInterpolator.java
+++ b/wres-config/src/wres/config/yaml/DeclarationInterpolator.java
@@ -905,6 +905,25 @@ public class DeclarationInterpolator
                                                                                      .stream() )
                                                                      .collect( Collectors.toSet() );
 
+        if ( dimensions.isEmpty() )
+        {
+            LOGGER.debug( "Interpolating a default dimension of {} for the summary statistics: {}.",
+                          SummaryStatistic.StatisticDimension.FEATURES,
+                          builder.summaryStatistics()
+                                 .stream()
+                                 .map( SummaryStatistic::getStatistic )
+                                 .toList() );
+
+            Set<SummaryStatistic> newStatistics = builder.summaryStatistics().stream()
+                                                         .map( s -> s.toBuilder()
+                                                                     .addDimension( SummaryStatistic.StatisticDimension.FEATURES )
+                                                                     .build() )
+                                                         .collect( Collectors.toSet() );
+            builder.summaryStatistics( newStatistics );
+
+            return;
+        }
+
         // Preserve iteration order of dimensions as they are listed - use a sorted set
         Set<SummaryStatistic.StatisticDimension> poolShaped =
                 dimensions.stream()
@@ -917,7 +936,8 @@ public class DeclarationInterpolator
                                         || s == SummaryStatistic.StatisticDimension.FEATURE_GROUP )
                           .collect( Collectors.toCollection( TreeSet::new ) );
 
-        if ( !poolShaped.isEmpty() && !featureShaped.isEmpty() )
+        if ( !poolShaped.isEmpty()
+             && !featureShaped.isEmpty() )
         {
             Set<SummaryStatistic> summaryStatistics = new HashSet<>();
 

--- a/wres-config/test/wres/config/yaml/DeclarationInterpolatorTest.java
+++ b/wres-config/test/wres/config/yaml/DeclarationInterpolatorTest.java
@@ -1342,6 +1342,60 @@ class DeclarationInterpolatorTest
     }
 
     @Test
+    void testInterpolateDimensionForSummaryStatisticsWhenNoneDeclared()
+    {
+        Set<SummaryStatistic> summaryStatistics = new LinkedHashSet<>();
+        SummaryStatistic mean = SummaryStatistic.newBuilder()
+                                                .setStatistic( SummaryStatistic.StatisticName.MEAN )
+                                                .build();
+        summaryStatistics.add( mean );
+
+        MetricParameters parameters =
+                MetricParametersBuilder.builder()
+                                       .summaryStatistics( summaryStatistics )
+                                       .build();
+        Metric first = MetricBuilder.builder()
+                                    .name( MetricConstants.TIME_TO_PEAK_ERROR )
+                                    .parameters( parameters )
+                                    .build();
+
+        Set<Metric> metrics = Set.of( first );
+
+        // Add summary statistics in two separate contexts
+        EvaluationDeclaration evaluation =
+                EvaluationDeclarationBuilder.builder()
+                                            .left( this.observedDataset )
+                                            .right( this.predictedDataset )
+                                            .metrics( metrics )
+                                            .summaryStatistics( summaryStatistics )
+                                            .build();
+
+        EvaluationDeclaration actual = DeclarationInterpolator.interpolate( evaluation, false );
+
+        Set<SummaryStatistic> actualOne = actual.summaryStatistics();
+        Set<SummaryStatistic> actualTwo = actual.metrics()
+                                                .stream()
+                                                .findFirst()
+                                                .orElseThrow()
+                                                .parameters()
+                                                .summaryStatistics();
+
+        SummaryStatistic expectedFirst = SummaryStatistic.newBuilder()
+                                                         .setStatistic( SummaryStatistic.StatisticName.MEAN )
+                                                         .addDimension( SummaryStatistic.StatisticDimension.FEATURES )
+                                                         .build();
+
+        SummaryStatistic expectedSecond = SummaryStatistic.newBuilder()
+                                                          .setStatistic( SummaryStatistic.StatisticName.MEAN )
+                                                          .addDimension( SummaryStatistic.StatisticDimension.TIMING_ERRORS )
+                                                          .build();
+
+        assertAll( () -> assertEquals( Set.of( expectedFirst ), actualOne ),
+                   () -> assertEquals( Set.of( expectedSecond ), actualTwo ) );
+
+    }
+
+    @Test
     void testInterpolateCombinedDimensionsForSummaryStatistics()
     {
         Set<SummaryStatistic> summaryStatistics = new LinkedHashSet<>();

